drivers/gles3/rasterizer_gles3.cpp

```diff
@@ -99,6 +103,9 @@ void RasterizerGLES3::begin_frame(double frame_step) {
 	GLES3::Utilities *utils = GLES3::Utilities::get_singleton();
 	utils->_capture_timestamps_begin();
 
+	// Disable OpenGL linear to sRGB conversion, because Godot will always do this conversion itself.
+	glDisable(GL_FRAMEBUFFER_SRGB);
```

Comment:
I would prefer to do this in `RasterizerGLES3::RasterizerGLES3()` if its not going to change at runtime.

Reply:
Thanks! I don't know how that didn't occur to me. :-) Moved there in latest push!